sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -172,19 +174,23 @@ case class DescribeDatabaseCommand(
   override def run(sparkSession: SparkSession): Seq[Row] = {
     val dbMetadata: CatalogDatabase =
       sparkSession.sessionState.catalog.getDatabaseMetadata(databaseName)
+    val allDbProperties = dbMetadata.properties
     val result =
       Row("Database Name", dbMetadata.name) ::
         Row("Description", dbMetadata.description) ::
-        Row("Location", CatalogUtils.URIToString(dbMetadata.locationUri)) :: Nil
+        Row("Location", CatalogUtils.URIToString(dbMetadata.locationUri))::
+        Row("Owner Name", allDbProperties.getOrElse(PROP_OWNER_NAME, "")) ::
+        Row("Owner Type", allDbProperties.getOrElse(PROP_OWNER_TYPE, "")) :: Nil
```

Comment:
Shall we have a consistent default value? `PrincipalType.USER` and '', which one is better?

Reply:
We are facing this issue because we support multiple hive version and the lowest one 0.12 does not support this. `PrincipalType.USER` might be better, <empty string> is here just because of hive 0.12

Reply:
As https://github.com/apache/spark/pull/26080#discussion_r353185129 mentioned, the metastore will always return `USER` for hive version above 0.12. if user use hive 0.12 the code here also suites.